webapp/commonComponents/tooltip.js

```diff
@@ -22,21 +22,16 @@ class Tooltip extends React.Component {
   }
 
   mouseEnter () {
-    const { messages, type = '' } = this.props
-
-    if (!(R.isEmpty(messages) || R.isNil(messages))) {
+    const { messages, content, show, type = '' } = this.props
 
+    if (show && (content || !(R.isEmpty(messages) || R.isNil(messages)))) {
```

Comment:
why do you need content props? I would always use messages, and if you are invoking it with just one message (which is your content) you could pass it as ```messages={['my message']}```